project/src/components/tabs/tab-content/tab-content-overview.tsx

```diff
@@ -0,0 +1,54 @@
+import { Movie } from '../../../types/movie';
+import { ratingLevel, movieScore } from '../../../constants';
+
+
+type TabContentOverviewProps = {
+  movie: Movie;
+}
+
+function TabContentOverview(props: TabContentOverviewProps): JSX.Element {
+
+  const { movie } = props;
+  const {
+    scoresCount,
+    rating,
+    description,
+    director,
+    starring } = movie;
+
+  const getRatingLevel = (score: typeof rating) => {
+    switch (true) {
+      case score >= movieScore.Bad && score < movieScore.Normal:
+        return ratingLevel[ratingLevel.Bad];
+      case score >= movieScore.Normal && score < movieScore.Good:
+        return ratingLevel[ratingLevel.Normal];
+      case score >= movieScore.Good && score < movieScore.VeryGood:
+        return ratingLevel.Good;
+      case score >= movieScore.VeryGood && score < movieScore.Awesome:
+        return ratingLevel.VeryGood;
+      case score === movieScore.Awesome:
+        return ratingLevel.Awesome;
```

Comment:
```suggestion
      case score < movieScore.Normal:
        return ratingLevel[ratingLevel.Bad];
      case score < movieScore.Good:
        return ratingLevel[ratingLevel.Normal];
      case score < movieScore.VeryGood:
        return ratingLevel.Good;
      case score < movieScore.Awesome:
        return ratingLevel.VeryGood;
      default:
        return ratingLevel.Awesome;
```

используем принцип "исключенного третьего" :), ну или шестого :) ведь если score положительный, то он и больше чем Bad